spec/mercure.md

```diff
@@ -51,8 +52,8 @@ interpreted as described in [@!RFC2119].
     Progressive Web App or a Mobile App, but can also be a server.
 
  *  Target: A subscriber, or a group of subscribers. A publisher is able to securely dispatch
-    updates to specific targets. Using an HTTP [@!RFC7230] or HTTPS [@!RFC2818] URL to identify
-    targets is **RECOMMENDED**.
+    updates to specific targets. The target **MUST** be identified by an URI [!@RFC3987] or by a
+    string. Using an HTTPS [@!RFC2818] or HTTP [@!RFC7230] URI is **RECOMMENDED**.
```

Comment:
```suggestion
    string. Using an HTTPS [@!RFC7230] or HTTP [@!RFC7230] URI is **RECOMMENDED**.
```

The `https` URI scheme is actually defined in https://tools.ietf.org/html/rfc7230#section-2.7.2